connector-packager/tests/jar_packager.py

```diff
@@ -33,11 +34,11 @@ def create_jar(source_dir, files, jar_filename, dest_dir):
     logging.debug("Start packaging " + jar_filename + " from " + str(abs_source_path) + "...")
 
     # if dest dir doesn't exist, then create it
-    if not os.path.exists(dest_dir):
-        os.makedirs(dest_dir)
+    if not dest_dir.exists():
+        dest_dir.mkdir()
         logging.debug("Creating destination directory " + str(dest_dir))
 
-    with ZipFile(os.path.join(dest_dir, Path(jar_filename)), "w") as jar:
+    with ZipFile(dest_dir / jar_filename, "w") as jar:
```

Comment:
I'm a bit concerned about the implicit '/' concat for file path since windows uses '\'. I believe os.path.join concats using the appropriate delimiter.

Reply:
The slash operator [works differently/cleverly with `Path` objects](https://docs.python.org/3/library/pathlib.html#operators).